### ISSUE TITLE
TS.MRANGE exclude keys with no samples from the response 

### ISSUE THREAD

**Issue by SutnuiBorshik** [RedisTimeSeries/RedisTimeSeries#547]
Hi. Is it possible to exclude keys that have no samples in requested time period from the TS.MRANGE response? 

Example:
```
TS.ADD key1 1601500004000 15 LABELS l1 val1 l2 val1
TS.ADD key1 1601500005000 33

TS.ADD key2 1601400001000 12 LABELS l1 val1 l2 val2

TS.ADD key3 1601500009000 5 LABELS l1 val1 l2 val3

TS.MRANGE 1601500003000 1601500005000 AGGREGATION sum 2000 FILTER l1=val1
```
we receive result
```
1) 1) "key1"
   2) (empty array)
   3) 1) 1) (integer) 1601500004000
         2) 48
2) 1) "key2"
   2) (empty array)
   3) (empty array)
3) 1) "key3"
   2) (empty array)
   3) (empty array)
```
I would like keys `key2` and `key3` to be excluded from the output. I understand that we can filter them out on client side, but when amount of keys with requested label reaches tens of thousands and we add `WITHLABELS` to the request, then the response object becomes massive which affects a response time. 

So in the end no matter how small time interval you select, you would end up with the list of all time series that have requested label, even though only few of them have samples in requested period. And since we are constantly adding new keys in our application the performance will become only worse with the time.

**Comment by danni-m** [comment]
Hi @szaydel and @dpipemazo, we are wondering if to make the filtering by default or add another flag like `WITHLABELS`. there's also a question about backwards compatibility, which will probably drive us to add a flag for this.

**Comment by dpipemazo** [comment]
@danni-m I would agree that it's likely best not to break backwards compatibility. A flag would be a great feature!

**Comment by K-Jo** [comment]
@filipecosta90 @danni-m @dpipemazo `SKIPEMPTY`?

**Comment by danni-m** [comment]
@K-Jo what about `IGNORE_EMPTY`? also we probably want to an underscore to be more consistent with the rest of the API.

**Comment by ashtul** [comment]
A module parameter to have a default would be convenient since some users would never want `EMPTY`.
